rnd/autogpt_builder/src/components/node-input-components.tsx

```diff
@@ -230,10 +239,24 @@ export const NodeGenericInputField: FC<{
           errors={errors}
           className={className}
           displayName={displayName}
+          connections={connections}
           handleInputChange={handleInputChange}
           handleInputClick={handleInputClick}
         />
       );
+    case "object":
```

Comment:
For unknown/complex object, it doesn't hurt to define it as key-value pair, the input will have field anyway (?)